gobblin-modules/gobblin-azkaban/src/main/java/org/apache/gobblin/azkaban/AzkabanJobLauncher.java

```diff
@@ -130,6 +130,11 @@ public AzkabanJobLauncher(String jobId, Properties props)
 
     HadoopUtils.addGobblinSite();
 
+    // Configure root metric context
+    List<Tag<?>> tags = Lists.newArrayList();
```

Comment:
needed for this pr?

Reply:
Yes. Currently, the RootMetricContext instance with empty tags is accidentally created before we apply the azkaban tags. The change is to move the creation with azkaban tags in the beginning.

Reply:
got it